yt_dlp/extractor/tiktok.py

```diff
@@ -150,3 +152,52 @@ def _real_extract(self, url):
             raise ExtractorError('This video is private', expected=True)
 
         raise ExtractorError('Video not available', video_id=video_id)
+
+
+class TikTokUserIE(InfoExtractor):
+    IE_NAME = 'tiktok:user'
+    _VALID_URL = r'(?!.*/video/)https?://www\.tiktok\.com/@(?P<id>[\w\._]+)'
+    _TESTS = [{
+        'url': 'https://www.tiktok.com/@corgibobaa?lang=en',
+        'playlist_mincount': 45,
+        'info_dict': {
+            'id': '6935371178089399301',
+        },
+        'skip': 'Cookies are required to download from TikTok user account.'
+    }, {
+        'url': 'https://www.tiktok.com/@meme',
+        'playlist_mincount': 593,
+        'info_dict': {
+            'id': '79005827461758976',
+        },
+        'skip': 'Cookies are required to download from TikTok user account.'
+    }]
+
+    def _real_extract(self, url):
+        id = self._match_id(url)
+        webpage = self._download_webpage(url, id)
+        user_id = re.search(r'\"id\":\"(?P<userid>\d+)', webpage)
+        if not user_id:
+            raise ExtractorError('Cookies are required to download from TikTok user account.', expected=True)
```

Comment:
raise_login_required

Reply:
Changed the message.
raise_login_required is not necessary now.